source/geod24/LocalRest.d

```diff
@@ -572,11 +576,24 @@ public final class RemoteAPI (API) : API
                 }
             }
 
-            immutable pretty = format("%s%s", method_name, OverloadParams.stringof);
 
-            // ensure it's used with API.method, *not* RemoteAPI.method which
-            // is an override of API.method. Otherwise mangling won't match!
-            enum mangled = getBestMatch!(__traits(getOverloads, API, method_name));
+            // special-case: no other overloads, and parameter list is empty:
+            // just select that one API method
+            alias Overloads = __traits(getOverloads, API, method_name);
+            static if (Overloads.length == 1 && OverloadParams.length == 0)
+            {
+                immutable pretty = method_name ~ Parameters!(Overloads[0]).stringof;
```

Comment:
I found a new bug here. If I do `immutable pretty = format("%s%s", method.name, Parameters!(Overloads[0]).stringof;);`, I get a segfault..

Reply:
You mean `method_name` right ?

Reply:
yes sorry